docs/Recipes.md

```diff
@@ -0,0 +1,67 @@
+# Recipes
+
+# Typescript
+You will need to install `ts-loader` for Haul to work with TypeScript.
+
+```yarn add -D ts-loader```
+
+This is a `webpack.haul.js` that works with TypeScript.
+```javascript
+module.exports = ({ platform }, { module, resolve }) => ({
+  entry: `./src/index.${platform}.tsx`,
+  module: {
+    ...module,
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader'
+      },
+      ...module.rules
+    ]
+  },
+  resolve: {
+    ...resolve,
+    extensions: ['.ts', '.tsx', `.${platform}.ts`, '.native.ts', `.${platform}.tsx`, '.native.tsx', ...resolve.extensions],
+  },
+});
+```
+
+
+However, if you want to use synthetic defaults imports,
+aka things like `import React from 'react'` instead of `import * as React from 'react'`
```

Comment:
So, being fairly new to Typescript, this was a gotcha for me too. This is a nice touch to have this included.